samples/nrf9160/serial_lte_modem/prj.conf

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+# General config
+#CONFIG_ASSERT=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=3
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_DNS_RESOLVER=y
+CONFIG_NET_IPV4=n
+CONFIG_NET_IPV6=n
+CONFIG_NET_UDP=n
+CONFIG_NET_TCP=n
+
+# BSD library
+CONFIG_BSD_LIBRARY=y
+CONFIG_BSD_LIBRARY_TRACE_ENABLED=y
+
+#
+# Use GPIO
+#
+CONFIG_GPIO=y
+CONFIG_GPIO_NRFX=y
+CONFIG_GPIO_NRF_INIT_PRIORITY=40
+CONFIG_GPIO_NRF_P0=y
+
+# UART interface
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_2_NRF_UARTE=y
+CONFIG_UART_2_NRF_FLOW_CONTROL=y
+
+# Stacks and heaps
+CONFIG_MAIN_THREAD_PRIORITY=7
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+CONFIG_LTE_NETWORK_MODE_LTE_M_GPS=y
+
+# GPS
+CONFIG_NRF9160_GPS=y
+CONFIG_NRF9160_GPS_LOG_LEVEL_DBG=y
+CONFIG_GPS_USE_EXTERNAL=y
+CONFIG_GPS_DEV_NAME="NRF9160_GPS"
+
+CONFIG_GPS_CONTROL=y
+CONFIG_GPS_CONTROL_FIRST_FIX_CHECK_DELAY=10
+CONFIG_GPS_CONTROL_FIX_CHECK_INTERVAL=30
+CONFIG_GPS_CONTROL_FIX_TRY_TIME=360
+CONFIG_GPS_CONTROL_PSM_DISABLE_ON_STOP=n
+
+# SNTP
+CONFIG_SNTP=y
+
+# MQTT lib
+CONFIG_MQTT_SOCKET_LIB=y
```

Comment:
Some early feedback - the upstream MQTT library (`CONFIG_MQTT_LIB`) is slightly updated copy of the downstream library (`CONFIG_MQTT_SOCKET_LIB`), and we plan to remove the downstream copy. Could you use the upstream version instead? You can check #391 to see what steps were needed to transition nRF Cloud library to use the upstream MQTT implementation.

Reply:
The project used to use upstream MQTT and later adapted to downstream MQTT_SOCKET. No problem I'll change back to previous one, which need to update code accordingly, too.

Reply:
Thanks and sorry for the confusion, I'm struggling with downstream MQTT copy removal for a way too long...